app/models/landing_page/block/hero.rb

```diff
@@ -3,21 +3,27 @@ module LandingPage::Block
   HeroImage = Data.define(:alt, :sources)
 
   class Hero < Base
-    attr_reader :image, :hero_content, :theme, :theme_colour
+    attr_reader :image, :theme, :theme_colour
 
     def initialize(block_hash, landing_page)
       super
 
       alt, sources = data.fetch("image").values_at("alt", "sources")
       sources = HeroImageSources.new(**sources)
       @image = HeroImage.new(alt:, sources:)
-      @hero_content = LandingPage::BlockFactory.build_all(data.dig("hero_content", "blocks"), landing_page)
       @theme = data.fetch("theme", "default")
       @theme_colour = data["theme_colour"]
     end
 
     def full_width?
       true
     end
+
+    def hero_content
+      block_content = data.dig("hero_content", "blocks")
+      return if block_content.blank?
+
+      @hero_content = LandingPage::BlockFactory.build_all(block_content, landing_page)
+    end
```

Comment:
Probably not super-critical, but is there ever a chance that hero_content might be called more than once in the view - do we want to memoize this (or make it a private method that's called to set the old attribute during initialisation?)

Reply:
Memoizing is a good call. I'll update and fix up.